src/core_modules/capture-core/components/DataEntries/SingleEventRegistrationEntry/DataEntryWrapper/NewEventDataEntryWrapper.component.js

```diff
@@ -0,0 +1,84 @@
+// @flow
```

Comment:
This whole file isn't new. The original of it is called `NewEvent/DataEntryWrapper/NewEventDataEntryWrapper.component.js`.Github dont recognise this as a incremental change of the file. So I hope you knowing that this is not a complete new component that I built but a copied version of an existing one, will help you review this file.